src/datasets/datasets.controller.ts

```diff
@@ -777,6 +777,8 @@ export class DatasetsController {
               }),
             );
           } else {
+            /* eslint-disable @typescript-eslint/no-unused-expressions */
+            // TODO: check the eslint error  "Expected an assignment or function call and instead saw an expression"
```

Comment:
@nitrosx @Junjiequan Please check if just suppressing the eslint-error is correct here.

Reply:
It's good, at least for now